src/ol/render/canvas/textreplay.js

```diff
@@ -310,6 +310,7 @@ ol.render.canvas.TextReplay.prototype.getImage_ = function(text, fill, stroke) {
     var fillState = this.textFillState_;
     var textState = this.textState_;
     var pixelRatio = this.pixelRatio;
+    var scale = this.textScale_;
```

Comment:
minor: a `pixelRatio * scale` variable could be created to avoid to compute the value multiple times